src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
 // write your code here
+// Create promise1
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+// Add success and error handlers for promise1
+promise1.then(() => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+}).catch(() => {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
```

Comment:
Follow 'DRY' principe (Don't Repeat Yourself). So create a separated function that render your messages